k8s/namespaces/money-claims/cmc-claim-store/demo.yaml

```diff
@@ -4,18 +4,12 @@ metadata:
   name: cmc-claim-store
   namespace: money-claims
 spec:
-  releaseName: cmc-claim-store
-  rollback:
-    enable: true
-    retry: true
   chart:
     git: git@github.com:hmcts/hmcts-charts
     ref: master
     path: stable/cmc-claim-store
   values:
```

Comment:
Same here and likely in the other {env}.yaml files that don't need a specific version

Reply:
Chart section removed from base chart due to different environments using different sources